src/clients/eos-propagate-uncertainty.cc

```diff
@@ -82,14 +83,21 @@ class CommandLine :
 
         std::string pmc_sample_directory;
 
+        std::string mcmc_sample_file;
+        unsigned mcmc_sample_min, mcmc_sample_max;
+        bool mcmc_prefer_prerun;
+
         std::vector<LogPriorPtr> priors;
 
         CommandLine() :
             config(PriorSampler::Config::Default()),
             parameters(Parameters::Defaults()),
             pmc_sample_min(0),
             pmc_sample_max(0),
-            pmc_sample_directory("/data")
+            pmc_sample_directory("/data"),
+            mcmc_sample_min(0),
+            mcmc_sample_max(0),
+            mcmc_prefer_prerun(false)
```

Comment:
Shouldn't this default to ```true```?

Reply:
I don't think so. The prerun by definition is expected to show bias. So if you have main run and prerun, the default should be to take samples from the main run.